solution/backend/resources/serializers/resources.py

```diff
@@ -1,31 +1,40 @@
 import re
 
-from django.urls import reverse
-from drf_spectacular.utils import extend_schema_field, OpenApiTypes
 from django.db.models import Q
+from drf_spectacular.utils import extend_schema_field
```

Comment:
Up here I removed OpenApiTypes , and reverse since neither are used anymore.  The rest I simply reordered.